### PR TITLE
Update SetMpGamerTagColour.md and SetMpGamerTagHealthBarColour.md

### PR DESCRIPTION
**Pull request by Immodicus**
Please let me know if there's anything you'd like me to change.

### CODE REVIEW

File: HUD/SetMpGamerTagColour.md [hunk 1]
```diff
@@ -6,15 +6,28 @@ aliases: ["0x613ED644950626AE"]
 
 ```c
 // 0x613ED644950626AE 0x7E3AA40A
-void SET_MP_GAMER_TAG_COLOUR(int gamerTagId, int flag, int color);
+void SET_MP_GAMER_TAG_COLOUR(int gamerTagId, int component, int hudColorIndex);
 ```
 
 ```
```
Comment (technetium-cfx) [inline HUD/SetMpGamerTagColour.md:12]: You can remove the codefence. This is only used for NativeDB imports.

File: HUD/SetMpGamerTagColour.md [hunk 2]
```diff
@@ -6,15 +6,28 @@ aliases: ["0x613ED644950626AE"]
...
-Ranges from 0 to 255. 0 is grey health bar, ~50 yellow, 200 purple.  
+Sets a gamer tag's component colour
 ```
 
 ## Parameters
-* **gamerTagId**: 
-* **flag**: 
-* **color**: 
+* **gamerTagId**: a gamerTagId obtained using  for example: https://runtime.fivem.net/doc/natives/?_0xBFEFE3321A3F5015
```
Comment (technetium-cfx) [inline HUD/SetMpGamerTagColour.md:17]: Please use markdown style links as such: `[text](link)`

File: HUD/SetMpGamerTagColour.md [hunk 3]
```diff
@@ -6,15 +6,28 @@ aliases: ["0x613ED644950626AE"]
...
+* **hudColorIndex**: a hud colour index, see the full list here: pastebin.com/d9aHPbXN  
 
+## Examples
+```cs
+Player player = Game.Player; //get our player
+int gamerTagId = CreateFakeMpGamerTag(player.Character.Handle, player.Name, false, false, null, 0); //create a gamer tag
+SetMpGamerTagColour(gamerTagId, 0, 129); //set component 0(GAMER_NAME) colour to 129(HUD_COLOUR_YOGA) 
+```
+
+```lua
```
Comment (technetium-cfx) [inline HUD/SetMpGamerTagColour.md:28]: We prefer Lua examples to come before C# (and JS)

File: HUD/SetMpGamerTagColour.md [hunk 4]
```diff
@@ -6,15 +6,28 @@ aliases: ["0x613ED644950626AE"]
...
+Sets a gamer tag's component colour
 ```
 
 ## Parameters
-* **gamerTagId**: 
-* **flag**: 
-* **color**: 
+* **gamerTagId**: a gamerTagId obtained using  for example: https://runtime.fivem.net/doc/natives/?_0xBFEFE3321A3F5015
+* **component**: a component id, see the full list here: https://docs.fivem.net/docs/game-references/gamer-tags/#components-list
+* **hudColorIndex**: a hud colour index, see the full list here: pastebin.com/d9aHPbXN
```
Comment (technetium-cfx) [inline HUD/SetMpGamerTagColour.md:19]: Color and colour? Stick to `color` 🙂

**Review by technetium-cfx — CHANGES_REQUESTED** [review]

**Comment by Immodicus** [comment]
Thank you for your feedback!

**Pull request closed without merging**